### PR TITLE
adding dimension and unit for dollar per MWh.

### PR DESCRIPTION
**Pull request by mjamei19**

### CODE REVIEW

**Comment by nicoleepp** [comment]
If it's possible can you think about adding some sort of test for the units you added. Even if it's just confirming they are set.

**Comment by mjamei19** [comment]
some simple tests for the defined units are added.

File: test/runtests.jl [hunk 1]
```diff
@@ -2,8 +2,11 @@ using Compat.Dates
 using Compat.Test
 using PowerSystemsUnits
 
-using Unitful: @u_str
+Unitful.register(PowerSystemsUnits)
```
Comment (ararslan) [inline test/runtests.jl]: Unitful is not yet defined at this point; this should come after `using Unitful`.

Comment (mjamei19) [reply test/runtests.jl]: oops! Should be fixed now.

**Pull request merged**